tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/mapper/AbstractPathValueMapper.java

```diff
@@ -55,12 +55,11 @@
     }
 
     /**
-     * Returns a new {@code IcuData} instance produced by post-processing a set of results
-     * generated by calling sub-class method {@link #addResults()}. This is the only method which
-     * need be directly invoked by the sub-class implementation (other methods are optionally used
-     * from within the {@link #addResults()} callback).
+     * Post-processes results generated by calling the subclass method {@link #addResults()}. This
+     * is the only method which need be directly invoked by the sub-class implementation (other
+     * methods are optionally used from within the {@link #addResults()} callback).
      */
-    final IcuData generateIcuData(String icuName, boolean hasFallback) {
+    final void addIcuData(IcuData icuData) {
```

Comment:
These mapper changes (here and below) are just to switch to taking and modifying the IcuData rather than creating and returning one. This is all part of the change to process the CLDR data just once to build a single data instance.